include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -9088,6 +9333,34 @@
         }
       ]
     },
+    {
+      "category" : "ValueEnum",
+      "kind" : "FPDenormMode",
+      "enumerants" : [
+        {
+          "enumerant" : "Preserve",
+          "value" : 0
+        },
+        {
+          "enumerant" : "FlushToZero",
+          "value" : 1
+        }
+      ]
+    },
+    {
+      "category" : "ValueEnum",
+      "kind" : "FPOperationMode",
+      "enumerants" : [
+        {
+          "enumerant" : "IEEE",
```

Comment:
Similarly, I think it's preferable to add:

     "capabilities" : [ "FunctionFloatControlINTEL" ],
     "version" : "None"